### ISSUE TITLE
Update Wiki Page 

### ISSUE THREAD

**Issue by kcoronel** [hackforla/CivicTechJobs#401]
### Overview
We need to update our team wiki so that it can be helpful for the sustainability of the project. 

### Action Items
- [x] Go through wiki and make edits 
- [x] Review Access the Data Wiki and make note of what pages or resources we would like to include in our wiki 
- [x] delete the contents of each page of the old wiki and add a link to the corresponding new wiki page
- [x] Share wiki with team

### Resources/Instructions
[Access the Data Wiki](https://github.com/hackforla/access-the-data/wiki)

**Comment by kcoronel** [comment]
From Bonnie on Slack on 5/12/23: 

When I go to the wiki, it says
PLEASE READ: This wiki is no longer used or updated. All files have been migrated over to our [GitHub Pages site](https://hackforla.github.io/CivicTechJobs/)! PLEASE REFER TO OUR GITHUB PAGES SITE FOR THE NEWEST INFORMATION. IF YOU NEED TO UPDATE THE WIKI, LOOK FOR THE FILE IN OUR [MKDOCS REPO](https://github.com/hackforla/CivicTechJobs/tree/main/mkdocs/docs).
So then I go to the team page on the new MK Docs wiki and find the team page https://hackforla.github.io/CivicTechJobs/misc/the-team/ it has the following list of people as PMs
Sharad Gaurav
Syed Danish Imran
So I know those are not the PMs anymore, so I go to the old wiki team page  https://github.com/hackforla/CivicTechJobs/wiki/The-Team,  it has updated PMS
Salima Yacoubi Soussane
Karen Coronel
I recommend that you

1. update the new wiki
2. delete the contents of each page of the old wiki and add a link to the corresponding new wiki page
3. If the page is too hard to keep updated, please point to https://www.hackforla.org/projects/civic-tech-jobs as the single source of truth.

Also, I don't think you should refer to the MKDocs hosted using GitHub pages as the GitHub Pages site because its confusing.  Also the text is so long that it actually adds confusion.
I recommend changing the language to say

**Comment by kcoronel** [comment]
From Ava: Want me to just delete the old wiki. It hasn't been used for a long time, and I don't think the team has much capacity to maintain it.

I think last time we had a discussion about it, the conclusion was that it only had a use for the dev team, and for everyone else, it's just too much overhead.

**Comment by kcoronel** [comment]
![CTJ-vertical-logo](https://github.com/hackforla/CivicTechJobs/assets/38144130/3b602e30-f10f-463d-9667-6535fe6b5a99)

**Comment by kcoronel** [comment]
On 7/7 - I went through and updated the MKdoc with new team information as well as edited instructions for how to join the project (how to reach out to the team and a link to the Roster Google Form). I also updated list of team members. Changes have not committed because they need to be reviewed and approved by someone other than me. I deleted the old Wiki as Ava suggested, but may decide to bring the titles back in the Old Wiki and link the new wiki page.
